packages/drivers/odsp-driver-definitions/src/resolvedUrl.ts

```diff
@@ -15,6 +15,23 @@ export interface IOdspUrlParts {
 	itemId: string;
 }
 
+/**
+ * Information about the file to be migrated.
+ * Providing these properties to the /snapshot api will trigger the migration of the file to Fluid format.
+ * @legacy
+ * @alpha
+ */
+export interface IIfMatchMigrationKind {
+	/**
+	 * The current eTag of the file to be migrated.
+	 */
+	eTag: string;
+	/**
+	 * The itemId of the file to be migrated.
+	 */
+	itemId: string;
```

Comment:
No need to have itemId here. Itemid info will be present inside the request url and not in the headers.